packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from "react";
+import Field, { FieldProps } from "./Field";
+import AceEditor from "react-ace";
+import "ace-builds/src-noconflict/mode-sql";
+import "ace-builds/src-noconflict/mode-javascript";
+import "ace-builds/src-noconflict/mode-python";
+import "ace-builds/src-noconflict/mode-yaml";
+import "ace-builds/src-noconflict/mode-json";
+import "ace-builds/src-noconflict/theme-textmate";
+
+export type Props = Omit<
+  FieldProps,
+  "value" | "onChange" | "options" | "multi" | "initialOption"
+> & {
+  syntax: string;
+  placeholder?: string;
+  currentValue: string;
```

Comment:
I would name this prop "value" to match what we use in other form components.

Reply:
Good call